python/chemiscope/input.py

```diff
@@ -103,6 +108,44 @@ def create_input(
     by comparing the numbers atoms and structures in the dataset to the
     length of provided list/np.ndarray.
 
+    It is possible to pass 2D properties to be plotted in the info bar by
+    passing a 3D numpy array of values having the shape `(n_samples, 1, n_parameters)`
+    and ensuring the corresponding parameter exists. The previous example becomes:
+
+    .. code-block:: python
+
+        properties = {
+            'cheese': {
+                'target': 'atom',
+                'values': np.zeros((300, 1, 4)),
```

Comment:
Instead of having this strange 3D hack, how about if there is a parameter field then we assume 2D properties and don't expand them? i.e. we add a flag to `_linearize` / call a different function for N-dimensional properties

Reply:
in this case, I think the most straightforward way is to check is parameters exist or not, and/or for parameters only for multidimensional properties. What do you think?

Reply:
You mean checking `properties[name]["parameters"]` right? If so, I agree this would be the most straightforward!

Reply:
yes that's what I meant